### PR TITLE
Upgrade bos-cli

### PR DESCRIPTION
**Pull request by Megha-Dev-19**

### CODE REVIEW

File: instances/devhub.near/widget/devhub/entity/post/Panel.jsx
Comment (Megha-Dev-19) [inline instances/devhub.near/widget/devhub/entity/post/Panel.jsx:566]: Note: changed only to test devhub deployment.

**Comment by Megha-Dev-19** [comment]
@race-of-sloths include

**Comment by race-of-sloths** [comment]
@Megha-Dev-19 Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
We are launching a survey "Who is faster? You vs SpaceX"

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/Megha-Dev-19?type=bot&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/Megha-Dev-19?type=bot&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/Megha-Dev-19)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command
</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/Megha-Dev-19)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

File: .github/workflows/deploy-dev-testnet.yml
Comment (petersalomonsen) [inline .github/workflows/deploy-dev-testnet.yml:33]: Can we use `latest` instead of a fixed version?

`curl --proto '=https' --tlsv1.2 -LsSf https://github.com/bos-cli-rs/bos-cli-rs/releases/latest/download/bos-cli-installer.sh | sh`

**Review by petersalomonsen — CHANGES_REQUESTED** [review]

**Comment by petersalomonsen** [comment]
@race-of-sloths score 3

**Comment by race-of-sloths** [comment]
### 🌟 Score recorded!

@petersalomonsen, thank you for scoring this pull request in the Race of Sloths!

File: .github/workflows/deploy-dev-testnet.yml
Comment (Megha-Dev-19) [reply .github/workflows/deploy-dev-testnet.yml:33]: oh yes, but I m concerned about one thing, if anything breaks in latest, it would effect us as well, and also I don't think the versions update that quickly. 
wdyt?

Comment (petersalomonsen) [reply .github/workflows/deploy-dev-testnet.yml:33]: I was thinking the other way, that if something is not working in the current version, we don't have to make changes in the DevHUB repo when the fix version is published. Of course we need then to trust that there is good regression test coverage then.. Maybe it's too risky?

I'll resolve this. We can go with 0.3.15 for now 👍

**Review by petersalomonsen — APPROVED** [review]

**Comment by race-of-sloths** [comment]
### ✅ PR is finalized! 
Your contribution is much appreciated with a final score of 3!
You have received 30 Sloth points for this contribution

**Pull request merged**
